var/spack/repos/spack_repo/builtin/packages/py_metatensor/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.python import PythonPackage
+from spack.package import *
+
+# Necessary to pin each version to libmetatensor
+VERSION_MAP = {
+    "0.1.14": "ee1f87bc045beadafa63cc0ccd0ebcf9c9e9fe8d619529d28775b0c22e0bbe19",
+    "0.1.13": "d43dfba8c6092de3e7bbe29dcf8fa3680462abd68947ece809befc9b0f6c1d56",
+}
+
+
+class PyMetatensor(PythonPackage):
+    """Python bindings for metatensor-core"""
+
+    homepage = "https://docs.metatensor.org"
+    pypi = "metatensor-core/metatensor_core-0.1.14.tar.gz"
+
+    import_modules = ["metatensor"]
+
+    maintainers("HaoZeke", "luthaf")
+    license("BSD-3-Clause", checked_by="HaoZeke")
+
+    extends("python")
+    depends_on("py-numpy", type=("run"))
```

Comment:
```suggestion
    depends_on("py-numpy", type=("build", "run"))
```
?

See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#dependencies